pkg/coderefs/coderefs.go

```diff
@@ -83,11 +75,6 @@ func Scan(opts options.Options) {
 		fatalServiceError(fmt.Errorf("could not retrieve flag keys from LaunchDarkly: %w", err), ignoreServiceErrors)
 	}
 
-	if len(flags) == 0 {
-		log.Info.Printf("no flag keys found for project: %s, exiting early", projKey)
-		os.Exit(0)
-	}
```

Comment:
this check was extraneous - remove it to reduce the number of exit paths in this function